multiversx_sdk_cli/dependencies/modules.py

```diff
@@ -157,7 +157,7 @@ def _post_install(self, tag: str):
         dependencies.install_module('golang')
 
         self.build_binary(tag, 'test')
-        self.make_binary_symlink_in_parent_folder(tag, 'test', 'mandos-test')
+        self.make_binary_symlink_in_parent_folder(tag, 'test', 'run-scenarios')
```

Comment:
How can we improve this github actions checks? I saw this in rust contracts PRs also - we should either make sure when we ask for reviews that we do not have a errors, or remove the checks if they are irrelevant. Here it's not really the case, but I saw PRs that were unreviewable...

Reply:
Noted!